cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -173,7 +173,7 @@ public void execute() {
             //// check if output flag is set
             if (null != this.output) {
                 CommandUtil.printError(this.errStream,
-                        "'-o' and '--output' flag is only supported for building a single ballerina file.",
+                        "The '-o' and '--output' flags are only supported when building a single Ballerina file.",
```

Comment:
"standalone Ballerina source file" instead of "single Ballerina file"?

Reply:
Changed to "standalone Ballerina file".